keyboards/dumbpad/dumbpad.h

```diff
@@ -0,0 +1,41 @@
+/* Copyright 2019 Chip
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+
+#include "quantum.h"
+
+bool encoder_update_user(uint8_t index, bool clockwise);
```

Comment:
```suggestion
bool encoder_update_user(uint8_t index, bool clockwise);
```

Reply:
Compiling fails without that line - I think the prototype is needed due to `encoder_update_kb` calling the function in dumbpad.c. I'll just move the `encoder_update_kb` call to the keymap.

Reply:
Resolved in d89cc21

Reply:
Oh, I see, you've changed the return type. TBH all of the keyboard/user hooks should be returning booleans instead of void, it makes it easier to do what you've done here (checking what the _user function did before running the _keyboard function).